build.sbt

```diff
@@ -1007,7 +1007,7 @@ lazy val `scio-smb`: Project = project
     description := "Sort Merge Bucket source/sink implementations for Apache Beam",
     libraryDependencies ++= Seq(
       "org.apache.beam" % "beam-sdks-java-core" % beamVersion,
-      "org.apache.beam" % "beam-sdks-java-core" % "it,test" classifier "tests",
+      "org.apache.beam" % "beam-sdks-java-core" % "2.16.0" classifier "tests",
```

Comment:
Looks like a typo in the original change? Surprised that it works. cc @ClaireMcGinty